scripts/install_binder.sh

```diff
@@ -16,6 +16,8 @@ usermod -l "${NB_USER}" "${DEFAULT_USER}"
 mkdir -p "${PYTHON_VENV_PATH}"
 chown -R "${NB_USER}" "${PYTHON_VENV_PATH}"
 
+PATH=/opt/pyenv/bin:${PATH}
+
 # And set ENV for R! It doesn't read from the environment...
 echo "PATH=${PATH}" >>"${R_HOME}/etc/Renviron.site"
```

Comment:
Hmm... I believe RStudio overrides system environmental variables not set in `Renviron.site`  -- I think we now already pass necessary system Renviron variables in the init system, though it's possible to run RStudio without relying on init.  But I haven't given that a close look recently.  

@eitsupi you also mentioned separately that you noticed the tex setup was overwriting rather than appending to PATH, maybe a separate bugfix for the next refactor?

Reply:
Related to that, since `install_binder.sh` can only be executed after `install_python.sh`, there is no need to write PATH to `Renviron` by `install_binder.sh` again.
I believe it is better to call `install_python.sh` from `install_binder.sh`.

Reply:
:+1: